src/main/java/build/buildfarm/common/ShardBackplane.java

```diff
@@ -295,4 +296,7 @@ boolean pollOperation(QueueEntry queueEntry, ExecutionStage.Value stage, long re
 
   @ThreadSafe
   Boolean validQueueProperties(List<Platform.Property> provisions);
+
+  @ThreadSafe
```

Comment:
seems like we make every method threadsafe here. do you know why?  we're only doing a read with jedis, but maybe it has something to do with only 1 jedis being used at a time?